src/NuGet.Services.BasicSearch/Startup.cs

```diff
@@ -168,7 +168,11 @@ public void Configuration(IAppBuilder app)
             ServicePointManager.DefaultConnectionLimit = Int32.MaxValue;
             ServicePointManager.Expect100Continue = false;
             ServicePointManager.UseNagleAlgorithm = false;
-            
+
+            // Ensure that SSLv3 is disabled and that Tls v1.2 is enabled.
```

Comment:
Is there some place in `ServerCommon` we can put all of this `ServicePointManager` bootstrapping stuff?